modules/cms/classes/CmsObject.php

```diff
@@ -9,7 +9,7 @@
 use ApplicationException;
 use ValidationException;
 use Exception;
-
+use Cms\Classes\Theme;
```

Comment:
Why is this necessary? The Theme class exists within this namespace.

Reply:
Definitely no reason - removed.